qcodes/utils/helpers.py

```diff
@@ -654,3 +654,16 @@ def inner(**inner_kwargs):
     inner.__doc__ = docstring
 
     return inner
+
+
+def abstractmethod(funcobj):
+    """A decorator indicating abstract methods.
+
+    This is heavily inspired by the decorator of the same name in
+    the ABC standard library. But we make our own version because
+    we actually want to allow the class with the abstract method to be
+    instantiated and we will use this property to detect if the
+    method is abstract and should be overwritten.
+    """
+    funcobj.__qcodesisabstractmethod__ = True
```

Comment:
snake case for readability? :)
```suggestion
    funcobj.__qcodes_is_abstract_method__ = True
```